deep_translator/mint.py

```diff
@@ -0,0 +1,110 @@
+"""
+google translator API
+"""
+
+__copyright__ = "Copyright (C) 2020 Nidhal Baccouri"
+
+from typing import List, Optional
+
+import requests
+import json
+
+from deep_translator.base import BaseTranslator
+from deep_translator.constants import BASE_URLS, WIKIMEDIA_MINT_LANGUAGE_TO_CODE
+from deep_translator.exceptions import (
+    RequestError,
+    TooManyRequests,
+    ServerException,
+    TranslationNotFound
+)
+from deep_translator.validate import is_empty, is_input_valid, request_failed
+
+
+class WikimediaMinTMachineTranslator(BaseTranslator):
+    """
+    class that wraps functions, which use Wikimedia Translate under the hood to translate text(s)
+    """
+
+    def __init__(
+        self,
+        source: str = "en",
+        target: str = "pl",
```

Comment:
why "pl"

Reply:
I have changed it to es(Spanish), though it was the default language whenever you try to translate.